twilight-http-ratelimiting/src/request.rs

```diff
@@ -162,6 +162,10 @@ pub enum Path {
     ChannelsIdTyping(u64),
     /// Operating on a channel's webhooks.
     ChannelsIdWebhooks(u64),
+    /// Operating on an application's entitlements.
+    ApplicationIdEntitlements(u64),
+    /// Operating on an application's SKUs.
+    ApplicationIdSKUs(u64),
```

Comment:
Is this path included in the path parsing? I don't see an arm for `["applications", id, "skus"]`.